src/GraphQL/Resolver/QueryType.php

```diff
@@ -438,10 +438,21 @@ static function ($fullpath) use ($db) {
                 return $db->quote($tag);
             }, $args['tags'])));
 
-            $conditionParts[] = "o_id IN (
+            $field = "name";
+            if (isset($args['useTagIdPath']) && $args['useTagIdPath']) {
+                $field = "idPath";
+            }
+
+            $statement = "o_id IN (
```

Comment:
please use `Service::getVersionDependentDatabaseColumnName` to be compatible with Pimcore versions 10 and 11 since the `o_` column is deprecated.